plugins-ui/src/modules/shared/wallet/vulticonnectWalletService.ts

```
@@ -0,0 +1,92 @@
+    if (!window.vultisig?.ethereum) {
+      alert(`No ethereum provider found. Please install VultiConnect.`);
+      return;
+    }
+
+    try {
+      const signature = await window.vultisig.ethereum.request({
+        method: "personal_sign",
+        params: [hexMessage, walletAddress],
+      });
+
+      if (signature && signature.error) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)